plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/local/LocalDockerModule.java

```diff
@@ -85,12 +82,14 @@ protected void configure() {
         devMachineVolumes.addBinding().toProvider(org.eclipse.che.plugin.docker.machine.ext.provider.ExtraVolumeProvider.class);
 
         Multibinder<Set<String>> networks = Multibinder.newSetBinder(binder(),
-                                                                     new TypeLiteral<Set<String>>() {},
```

Comment:
Please do not change formatting here and [there ](https://github.com/eclipse/che/pull/4331/files#diff-3ebfd325784cb24ec99509e5d4ebc569R91) unless you have strong opinion on that